hanjaelee/services/userService.js

```diff
@@ -16,10 +17,46 @@ const signup = async (name, email, password, profileImage) => {
   return isCreated
 }
 
-const makeHashedPassword = (password, saltRounds) => {
-  return bcrypt.hash(password, saltRounds)
+const login = async (email, password) => {
+  try {
+    let accessToken = null
+
+    const userId = await userDao.getUserId(email)
+    const hashedPassword = await userDao.getPassword(userId)
+    if (!userId || !password) return accessToken
+
+    const isSame = await checkHashedPassword(password, hashedPassword)
+    if (!isSame) return accessToken
+
+    const current = getCurrentTimeInSeconds()
+    const payLoad = {
+      iss: 'Hanjae Lee',
+      iat: current,
+      exp: current + (60 * 60 * 24),
```

Comment:
expire의 경우 현재시간보다 이후에 존재하여야 합니다. 이 부분에서 `current`를 함수로 분리하여
적용하셨군요! 너무 잘해주셨습니다! 만료시간을 바로 적는 것보다 상수로 만들어 적용해 보세요!
```suggestion
      exp: current + EXPIRE_TIME,
```